docs/visualization/path/response_time.md

```diff
@@ -87,3 +87,44 @@ plot.show()
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
+
+## TimeSeries
+
+The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the best case, worst case, worst_in_input case, or all case.
+
+```python
+# plot best case
+plot = Plot.create_response_time_timeseries_plot(path)
+plot.show()
+# or
+# plot = Plot.create_response_time_timeseries_plot(path, case='best')
+# plot.show()
+```
+
+![response_time_timeseries_best](../../imgs/response_time_timeseries_best.png)
+
+```python
+# plot worst case
+plot = Plot.create_response_time_stacked_bar_plot(path, case='worst')
+plot.show()
+```
+
+![response_time_timeseries_worst](../../imgs/response_time_timeseries_worst.png)
+
+```python
+# plot worst_in_input case
+plot = Plot.create_response_time_stacked_bar_plot(path, case='worst_in_input')
```

Comment:
```suggestion
plot = Plot.create_response_time_stacked_bar_plot(path, case='worst-in-input')
```

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/215/commits/0759fa70312403a38e6333b5c059df59af3d1124.